src/components/icons.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { SVGProps } from 'react';
+
+type Icon = SVGProps<SVGSVGElement>;
+
+export enum Color {
+  Active = '#0ba8e0',
+  Inactive = '#7a8287',
+}
+
+export const ThumbsUpIcon = (props: Icon): JSX.Element => (
```

Comment:
no need to specefically set the return type
```suggestion
export const ThumbsUpIcon = (props: IconProps) => (
```